prow/cmd/tide/config.md

```diff
@@ -106,6 +107,17 @@ tide:
   target_url: https://prow.k8s.io/tide
 
   queries:
+  - orgs:
+    - kubeflow
+    excludedRepos:
+    - kubeflow/community
+    - kubeflow/examples
+    labels:
```

Comment:
there's already examples on labels and missingLabels. I think you just need to line 112 and 113. since `kubeflow/community` and `kubeflow/examples` have been used in `repos`, could you exclude `kubeflow/website` as an example?

Reply:
You do have a point. I will exclude `kubeflow/website` to delineate which repos won't be queried by tide.
However, line 112 to 120 is to explain one function of `excludedRepos` that you could configure distinguish query condition of specific repositories in `repos` which is different from other repos in the orgs.

Reply:
Update, I've add `kubuflow/website` as an example of how tide ignore querying specific repository.